content/md/imports-exports/tailored-import.md

```diff
@@ -198,6 +198,16 @@ This operation is available for `Text` and `Text Area` attributes used as a targ
 It will help you to search for a term and replace it by another one. Note that you can add up to 10 replaced values per operation.
 ![Search & replace operation](../img/Tailored-Import_Operation_Search-and-Replace.png)
 
+## Extract with regular expression
+
+This operation is available for `Text` and `Text Area` attributes used as a target.
+It will help you to extract a value based on a regular expression from your choice.
+![Regular expression operation](../img/Tailored-Import_Operation_Regular-expression.png)
+
+::: tips
+Your regular expression shoud start and end by a slash (/).  
```

Comment:
It's not true it can also be a ~. So for me we just redirect user to the regex documentation
```suggestion
```